CHANGELOG.md

```diff
@@ -4,13 +4,39 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Common Changelog](https://common-changelog.org).
 
+[1.2.0]: https://github.com/sablier-labs/v2-periphery/compare/v1.1.1...v1.2.0
 [1.1.1]: https://github.com/sablier-labs/v2-periphery/compare/v1.1.0...v1.1.1
 [1.1.0]: https://github.com/sablier-labs/v2-periphery/compare/v1.0.3...v1.1.0
 [1.0.3]: https://github.com/sablier-labs/v2-periphery/compare/v1.0.2...v1.0.3
 [1.0.2]: https://github.com/sablier-labs/v2-periphery/compare/v1.0.1...v1.0.2
 [1.0.1]: https://github.com/sablier-labs/v2-periphery/compare/v1.0.0...v1.0.1
 [1.0.0]: https://github.com/sablier-labs/v2-periphery/releases/tag/v1.0.0
 
+## [1.2.0] - 2024-07-01
+
+### Changed
+
+- Rename `MerkleStreamer` to `MerkleLockup` ([#268](https://github.com/sablier-labs/v2-periphery/pull/268))
+- Rename `Batch` to `BatchLockup` ([#322](https://github.com/sablier-labs/v2-periphery/pull/322),
+  [#297](https://github.com/sablier-labs/v2-periphery/pull/297),
+  [#300](https://github.com/sablier-labs/v2-periphery/pull/300))
```

Comment:
why are #297 and #300 added at this point?

wouldn't #322 be suffice?

Reply:
Yeah.